dali/kernels/imgproc/paste/paste.h

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_IMGPROC_PASTE_PASTE_H_
+#define DALI_KERNELS_IMGPROC_PASTE_PASTE_H_
+
+#include <utility>
+#include "dali/util/ocv.h"
+#include "dali/core/common.h"
+#include "dali/core/convert.h"
+#include "dali/core/geom/box.h"
+#include "dali/core/error_handling.h"
+#include "dali/kernels/kernel.h"
+#include "dali/kernels/imgproc/roi.h"
+
+const int Y_AXIS = 0;
+const int X_AXIS = 1;
+const int C_AXIS = 2;
+
+namespace dali {
+namespace kernels {
+
+template<typename OutputType, typename InputType>
+class PasteCpu {
+ public:
+  using Image = InTensorCPU<InputType, 3>;
+  using OutImage = OutTensorCPU<OutputType, 3>;
+  using Coords = InTensorCPU<const int, 1>;
+
+  KernelRequirements
+  Setup(KernelContext &context, const OutTensorCPU<OutputType> &pasteFrom, const Coords &in_anchors,
+        const Coords &in_shapes, const Coords &out_anchors) {
+    // Kernel is not aware of the size of the output. This should be handled in the operator.
+    return nullptr;
+  }
+
+
+  /**
+   * Assumes HWC memory layout
```

Comment:
Nitpick: The first line will be displayed as a brief description of the function.
```suggestion
   * Pastes regions of inputs onto the output.
   * 
   * @remarks Assumes HWC memory layout.
```